signal_client_service: "/bid_in_European_roulette"
topic_reading_answer: "/anser_to_topic"

